tests/console/suseconnect_scc.pm

```diff
@@ -39,7 +39,8 @@ sub run {
 
     select_console('root-console');
     assert_script_run $cmd;
-    assert_script_run 'SUSEConnect --list-extensions';
+    assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'    \e\[1mServer Applications Module\')"';
+    assert_script_run 'SUSEConnect --list-extensions | grep "$(echo -en \'    \e\[1mWeb and Scripting Module\')"';
     assert_screen 'activated-with-suseconnect';
```

Comment:
I would move  this line directly after the `assert_script_run 'SUSEConnect --list-extensions';` line to check for the original using the needles - not for the output of grep.

Reply:
okay, thanks